LanguageFeatures/Augmentation-libraries/augmented_expression_A01_t19_lib.dart

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion The exact result of an `augmented` expression depends on what is
+/// being augmented, but it generally follows the same rules as any normal
+/// identifier:
+/// - Augmenting getters: Within an augmenting getter `augmented` invokes the
+///   getter and evaluates to the return value. If augmenting a field with a
+///   getter, this will invoke the implicit getter from the augmented field.
+///
+/// @description Checks that it is a compile-time error to if an augmenting
+/// getter has return type with the name `augmented`.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=macros
+
+augment library 'augmented_expression_A01_t19.dart';
+
+augment String get topLevelGetter {
+  augmented? x;
```

Comment:
How would this help testing augmenting function declarations where a parameter has a type whose name is `augmented`?

Reply:
It was just a typo caused by a copy-paste. The test checka use of a type with the name `augmented` in a body of a getter